.evergreen.yml

```diff
@@ -15,15 +15,15 @@ mongo_tools_variables:
       mongod_args: ""
       mongod_port: 33333
     ssl: &mongod_ssl_startup_args
-      mongod_args: "--sslMode requireSSL --sslCAFile db/testdata/ca.pem --sslPEMKeyFile db/testdata/server.pem"
+      mongod_args: "--sslMode requireSSL --sslCAFile db/testdata/ca-ia.pem --sslPEMKeyFile db/testdata/test-server.pem"
       mongod_port: 33333
 
   mongo_arguments:
     default: &mongo_default_startup_args
       mongo_args: &mongo_default_startup_args_string "--port 33333"
       mongod_port: 33333
     ssl: &mongo_ssl_startup_args
-      mongo_args: "--port 33333 --ssl --sslCAFile ./db/testdata/ca.pem --sslPEMKeyFile ./db/testdata/server.pem --sslAllowInvalidCertificates"
```

Comment:
We should not use --sslAllowInvalidCertificates in the test, since there is a bug when this flag is set, we skip the certificate verification at all. I created ticket https://jira.mongodb.org/browse/TOOLS-2587 to track this problem. 
Here I created a certificate with `localhost` as hostname to avoid this flag.